src/services/marzipano/marzipano.js

```diff
@@ -0,0 +1,144 @@
+import Marzipano from 'marzipano';
```

Comment:
Could we split this file into smaller bits? Perhaps make it a bit more reusable, it's doing a LOT of things here 😛  Also noticed a Todo, some eslint-disables and some arbitrary values in createHotspotTemplate. Also the addEventListener isn't being removed anywhere... Potential memory leak?

Reply:
out of scope